app/src/processing/app/Base.java

```diff
@@ -1367,7 +1367,7 @@ public void rebuildRecentBoardsMenu() throws Exception {
               buttonGroupsMap,
               board, board.getContainerPlatform(), board.getContainerPlatform().getContainerPackage());
       boardMenu.insert(item, 3);
-      item.setAccelerator(KeyStroke.getKeyStroke('0' + index,
+      item.setAccelerator(KeyStroke.getKeyStroke('1' + index,
```

Comment:
This seems an unrelated off-by-one bugfix that should have been squashed into the previous commit?

Reply:
Not an off by one really but a different shortcut, so I'd leave it as a separate commit.

Reply:
I mean that this shortcut is introduced in the previous commit and changed in this commit, so there is not really a point in separating them? You might as well introduce the shortcut correctly in the first commit? Also, this change (short 0 to 1) is currently in the " Fix concurrent access to menuItemsToClickAfterStartup" commit, where it does not belong at all AFAICS? So it should at least be moved into its own commit then.